test/appearance/appearance_assertion.rb

```diff
@@ -0,0 +1,13 @@
+require 'test/unit'
+
+module AppearanceAssertion
+  include Test::Unit::Assertions
+
+  def assert_same_image(expected_image_path, image_object, delta = 0.01)
```

Comment:
For the image above, should the delta be `0.0`?

Reply:
Also, I wonder if we should leave the `delta` argument out for now, until we need it.

Reply:
There seems to be a subtle error between the ImageMagick versions.

Reply:
I see. In that case, should we make `delta` a keyword argument? `delta: 0.01`.

Reply:
OK I see.